aleccunningham/ispa
# look at these fucking commits

they're magic
- no more members, attendees
- symbol dict to dynamically create objects
- factory_boy, pytest and fake for great testing 
- polished docker-compose and travisci
- removal of api app, and creating viewsets inside events app
- add to_json and  create_model to every model
- use sqlite when testing, postgres for everything else
- ./manage.py fakefixtures django command that uses factory_boy to generate:
50 events
50 locations
50 users
50 owners
50 attendees
- you can use ./manage.py loaddata fixtures/v01-alpha3.json, thats the db we test on
- make sure manage.py loads settings/test.py, and ./local anytime else
- event model has 2 natives m2m keys instead of the modelcluster crutch--owner and attendee
- owner and attendee are both 'through' models (still need to create user proxy)

**update** API implemented for /events/, /locations/, /users/, and registration (all in the readme)
